docs/pages/desktop-access/getting-started.mdx

```diff
@@ -1,199 +1,237 @@
-  auth_token: /tmp/token
-windows_desktop_service:
-  enabled: yes
-  non_ad_hosts:
-    - 192.0.2.156
-auth_service:
-  enabled: no
-proxy_service:
-  enabled: no
-ssh_service:
-  enabled: no
-```
+   ```code
+   $ tctl tokens add --type=windowsdesktop
+   ```
 
-Note that without Active Directory, Teleport cannot automatically discover your
-Desktops. Instead you must define the Windows systems configured for access through
-Teleport in your config file, or use Teleport's [API](../api/introduction.mdx)
-to build your own integration. An example API integration is available on
-[GitHub](https://github.com/gravitational/teleport/tree/master/examples/desktop-registration).
-
-<Details title="Add labels to hosts">
-You can attach labels to your Windows hosts by matching to their hostnames.
-For example, to add the `cloud: ec2` label to hosts with EC2 private IP DNS names:
-
-```diff
-version: v3
-teleport:
-  nodename: windows.teleport.example.com
-  proxy_server: teleport-proxy.example.com:443
-windows_desktop_service:
-  enabled: yes
-  non_ad_hosts:
--    - 192.0.2.156
-+    - ip-192-0-2-156.us-east-2.compute.internal
-+ host_labels:
-+    - match: '.*\.us-east-2.compute.internal'
-+      labels:
-+        cloud: ec2
-auth_service:
-  enabled: no
-proxy_service:
-  enabled: no
-ssh_service:
-  enabled: no
-```
+   The command displays information similar to the following:
+
+   ```text
+   The invite token: (=presets.tokens.first=)
+   This token will expire in 60 minutes.
+   ```
+
+1. Copy the token from your administrative workstation and paste it into a file on the 
+Linux server where you want to run the Windows Desktop Service.
+
+   For example, create a file named `/tmp/token` on the Linux server and paste the token
+   into the file.
+
+1. Check whether Teleport is installed on the Linux server where you want to run the 
+Windows Desktop Service by running the following command:
+
+   ```code
+   $ teleport version
+   ```
+
+   If Teleport isn't installed on the Linux server, follow the appropriate [Installation
+instructions](../installation.mdx#installation-instructions) for your environment.
+
+1. Configure the `/etc/teleport.yaml` file on the Linux server with settings similar to the 
+following for the Windows Desktop Service: 
 
-</Details>
+   ```yml
+   version: v3
+   teleport:
+     nodename: <Var name="windows.teleport.example.com" />
+     proxy_server: <Var name="teleport.example.com:443" />
+     auth_token: <Var name="/tmp/token" />
+   windows_desktop_service:
+     enabled: yes
+     non_ad_hosts:
+     - 192.0.2.156
+   auth_service:
+     enabled: no
+   proxy_service:
+     enabled: no
+   ssh_service:
+     enabled: no
+   ```
 
-(!docs/pages/includes/start-teleport.mdx service="the Teleport Desktop Service"!)
+   In this file:
 
-## Step 3/4. Configure Windows access
+   - Set the `proxy_server` to the address of your Teleport cluster.
+   - List the Windows IP addresses or DNS names under `non_ad_hosts`.
```

Comment:
```suggestion
   - List the Windows IP addresses or hostnames under `non_ad_hosts`.
```

DNS is the mechanism by which a hostname is resolved to an IP address, so I would prefer using hostname over "DNS name"